Source/WebCore/dom/Document.cpp

```diff
@@ -713,7 +713,7 @@ Document::~Document()
 
     ASSERT(!renderView());
     ASSERT(m_backForwardCacheState != InBackForwardCache);
-    ASSERT(m_ranges.isEmpty());
+    ASSERT(m_ranges.isEmptyIgnoringNullReferences());
```

Comment:
This assertion is randomly crashing.
https://bugs.webkit.org/show_bug.cgi?id=253765#c4